src/window/action_popup.c

```diff
@@ -134,18 +112,31 @@ static void draw_option_image_border(int x, int y, int focused)
 static void draw_foreground(void)
 {
     graphics_in_dialog();
+
     if (data.option_1.image_id) {
         draw_option_image_border(120, 140, data.focus_button_id == 1);
     }
+
+    text_draw_multiline(translation_for(data.option_1.header), 260, 150, 270, data.selected_policy == 1 ? FONT_NORMAL_WHITE : FONT_NORMAL_BLACK, 0);
```

Comment:
Your "white on black" issue is caused by the fact that while you redraw the text, you don't redraw the banel background. So the text is written over itself and as the black text is slightly bigger, it shows behind the white text.

See below for a way to fix this (which involves putting all this text on `draw_background` again).

Reply:
Ok